pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools-rust"]
+
+[tool.isort]
+profile = "black"
+
+[tool.maturin]
+include = [
+    { path = "Cargo.lock", format = "sdist" }
+]
+exclude = [".github/**", "ci/**", ".asf.yaml"]
+# Require Cargo.lock is up to date
+locked = true
```

Comment:
Noting that I've been thinking about consolidating as much of our project metadata into pyproject.toml as possible (similar to other RAPIDS/Dask projects) which would make this file less stubby

Reply:
+1 to this. I've also been thinking of simplifying the setup.py scripts and moving as much as possible into the pyproject.toml file